src/components/20-molecules/datepicker/ui.test.js

```diff
@@ -68,22 +70,48 @@ test('should select the 31th of January from within the February view', async t
 test('should have a 29th of February in 2020 - should correctly handle leap year', async t => {
   const datePickerAccessor = new DatePickerAccessor(t, 'datepicker');
 
-  await datePickerAccessor.chooseFebruary();
+  await datePickerAccessor.chooseAnyMonth(2);
   await datePickerAccessor.selectDayOfCurrentMonth(29);
 
   await datePickerAccessor.assertYear(2020);
   await datePickerAccessor.assertMonth('Februar');
   await datePickerAccessor.assertDay(29);
 });
 
+test('should begin with September 1 as first Monday in 1980', async t => {
+  const datePickerAccessor = new DatePickerAccessor(t, 'datepicker');
+  await datePickerAccessor.chooseAnyMonth(9, false);
+  await datePickerAccessor.chooseAnyYear(1980, false);
+
+  await datePickerAccessor.assertYear(1980);
+  await datePickerAccessor.assertMonth('September');
+
+  const firstVisibleDay = await Selector(() =>
+    document.querySelector(
+      `axa-datepicker[data-test-id="datepicker"] .js-datepicker__calendar .js-datepicker__calender-body__cell:first-child`
+    )
+  );
+  await t.expect(firstVisibleDay.exists).ok();
+  await t.expect(firstVisibleDay.innerText).eql('1');
+});
+
+test('should handle month change with native dropdown element', async t => {
+  const datePickerAccessor = new DatePickerAccessor(t, 'datepicker');
+
+  await datePickerAccessor.chooseAnyMonth(2, true);
+  await datePickerAccessor.assertMonth('Februar');
```

Comment:
This test doesnt have a explicit `expect`. Its not perfectly readable. Probably renaming the method to `expectMonth` would be a bit better?!

Reply:
As discussed, we would clean up and 'flatten'  ui.test.js separately, since this is just one example of a bigger issue (missing stacktraces from Testcafe when expectations fail in external test functions like the one you mentioned).